modules/mobile/app/controllers/mobile/application_controller.rb

```diff
@@ -45,6 +45,7 @@ def authenticate
 
       session_manager = IAMSSOeOAuth::SessionManager.new(access_token)
       @current_user = session_manager.find_or_create_user
+      map_logingov_to_idme # check method comment below for more info
```

Comment:
Style preference: I think you could leave this comment out. If anyone wants to know what's going on, they'll look for the method declaration.